src/components/ComputeModal.js

```diff
@@ -664,13 +691,16 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
       const { computeZone, computeRegion } = getRegionInfo(location || defaultLocation, locationType)
       const runtimeConfig = currentRuntimeDetails?.runtimeConfig
       const gpuConfig = runtimeConfig?.gpuConfig
+      const autopauseConfig = currentRuntimeDetails ? { enabled: currentRuntimeDetails?.autopauseEnabled, threshold: currentRuntimeDetails?.autopauseThreshold } : null
```

Comment:
We don't need to use optional chaining for `currentRuntimeDetails` since we know it isn't `undefined` if we're in the first part of the ternary.

When we use a non-boolean (e.g. `currentRuntimeDetails`) as a boolean value, we use `!!` to coerce for explicitness about the intention.

Also, applied formatting.

```suggestion
      const autopauseConfig = currentRuntimeDetails ?
        { enabled: currentRuntimeDetails?.autopauseEnabled, threshold: currentRuntimeDetails?.autopauseThreshold } :
        null
```